src/flushMicroTasks.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import { printDeprecationWarning } from './helpers/errors';
+import { setImmediate } from './helpers/getTimerFuncs';
```

Comment:
Not sure about this change. Should promises always be flushed through real timers? Where does this flush even get called?

Reply:
@thymikee do you know more about where this flush is used and if it should always use real timers?